src/lib/Input.tsx

```diff
@@ -57,7 +59,7 @@ const Input = <T extends FieldValues>(props: InputProps<T>) => {
   const { id } = useSafeNameId(props.name, props.id);
 
   return (
-    <FormGroupLayout {...props} layout={formGroupLayout}>
+    <FormGroupLayout {...props} layout={formGroupLayout} inputGroupStyle={inputGroupStyle}>
```

Comment:
```suggestion
    <FormGroupLayout {...props} layout={formGroupLayout}>
```

Since the props are spread at the beginning you don't need to pass it explicitly.